src/installer/utils.py

```diff
@@ -93,6 +93,14 @@ def parse_metadata_file(contents: str) -> Message:
     return feed_parser.close()
 
 
+def normalize_distribution_name(name: str) -> str:
```

Comment:
Hmm... this might be easier to parse for folks if this has the same name as the `packaging` function.

```suggestion
def canonicalize_name(name: str) -> str:
```